demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -54,6 +59,32 @@ function ProductsGridToolbar() {
             <ToolbarItem>
                 <GridColumnsButton />
             </ToolbarItem>
+            <ToolbarItem>
+                <Button
+                    variant="text"
+                    ref={moreMenuRef}
+                    onClick={() => {
+                        setShowMoreMenu(true);
+                    }}
+                    endIcon={<MoreVertical />}
+                >
+                    <FormattedMessage id="products.moreActions" defaultMessage="Mehr Aktionen" />
```

Comment:
english, that could even be a common message exported by coment (not part of this PR)